cmd/lakectl/cmd/dbt_util_test.go

```diff
@@ -143,8 +143,8 @@ func TestDbtLsToJson(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.Name, func(t *testing.T) {
 			dbtResources, err := cmd.DBTLsToJSON(tt.Args.ProjectRoot, tt.Args.ResourceType, tt.Args.SelectValues, tt.Args.Executor.ExecuteCommand)
-			if (err != nil) && !errors.As(err, &tt.ErrType) {
-				t.Errorf("DBTLsToJSON() error = %v, expected ErrType: %v", err, tt.ErrType)
+			if err != nil && reflect.TypeOf(err) != reflect.TypeOf(tt.ErrType) {
```

Comment:
Why does `errors` package not good here?

Reply:
The call for `errors.As` expect to have a type it should map `err` into - in this case the type for all tests is `error`.
I assumed the test writer wanted to match the error type as suggested by the name 'ErrType'.